src/components/footer/footerListArray.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+
+export const FooterListArray = ({ header, links }) => {
+  const [arrow, setArrow] = useState(false);
+  const arrowHandler = () => {
+    setArrow(!arrow);
+  };
+  const arrowUp = <i className="fa fa-sort-up" onClick={arrowHandler}></i>;
+  const arrowDown = <i className="fa fa-sort-down" onClick={arrowHandler}></i>;
+
+  const listArray = links.map((listLink, index) => {
+    return (
+      <li className="propertyArea" key={index}>
+        <a href="#" className="liLink">{listLink}</a>
```

Comment:
What is liLink? You may want to change this class name to something like "footer-link"